configurations/performance/latency-decorator-error-thresholds-nemesis-ent.yaml

```diff
@@ -0,0 +1,51 @@
+latency_decorator_error_thresholds:
+  write:
+    _mgmt_repair_cli:
+      duration:
+        fixed_limit: 10800
+    _terminate_and_wait:
+      duration:
+        fixed_limit: 500
+    add_new_nodes:
+      duration:
+        fixed_limit: 3600
```

Comment:
It's per cycle ? right ?

Not total,
 i.e. maybe elasticity case and regular ops would need different numbers

Reply:
yes, per cycle. For elasticity we can define differently.